flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendTestBase.java

```diff
@@ -152,16 +137,15 @@
  * <p>NOTE: Please ensure to close and dispose any created keyed state backend in tests.
  */
 @SuppressWarnings("serial")
+@ExtendWith(ParameterizedTestExtension.class)
 public abstract class StateBackendTestBase<B extends AbstractStateBackend> extends TestLogger {
```

Comment:
`TestLogger` is a JUnit4 rule/extension
Here should be used `META-INF/services/org.junit.jupiter.api.extension.Extension`